experimental/examples/tree-sample/inventory-app/src/views/main.tsx

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ISharedTree } from "@fluid-internal/tree";
+import * as React from "react";
+import { useTree } from "@fluid-experimental/tree-react-api";
+import { Inventory } from "../domains/model";
+import { Counter } from "./counter";
+
+export const MainView: React.FC<{ tree: ISharedTree }> = ({ tree }) => {
+	const inventory = useTree<Inventory>(tree);
+
+	const counters = Object.keys(inventory).map((key) => {
+		const value = inventory[key] as number;
+
+		return (
+			<Counter
```

Comment:
React is probably mad that you don't have an `id` for these:
```suggestion
			<Counter
				id={key}
```